examples/todoapp/lib/base/models/request_models/push_notification_data_request_model.dart

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) 2023, Prime Holding JSC
+// https://www.primeholding.com
+//
+// Use of this source code is governed by an MIT-style
+// license that can be found in the LICENSE file or at
+// https://opensource.org/licenses/MIT.
+
+import 'package:json_annotation/json_annotation.dart';
+
+part 'push_notification_data_request_model.g.dart';
+
+@JsonSerializable()
+class PushNotificationDataRequestModel {
```

Comment:
As agreed all generated by the rx_bloc_cli files, which are not used need to be removed

Reply:
Done